.github/ISSUE_TEMPLATE/new-release.md

```diff
@@ -38,6 +38,10 @@ Please do not remove items from the checklist
 - [ ] Add a link to the tagged release in this issue.
 - [ ] Send an announcement email to `kubernetes-dev@googlegroups.com` with the subject `[ANNOUNCE] node-feature-discovery $VERSION is released`
 - [ ] Add a link to the release announcement in this issue
+- [ ] For a major release, update README in master branch
```

Comment:
what about a minor release ? e.g once v0.8.0 is released, say we need to release `v0.8.1` what will update helm chart in this case ?

Reply:
Haven't had any, yet 🤞  But yes, a better solution would be nice

Reply:
Found out that we can actually trigger on publish release event 🤦 
https://github.com/kubernetes-sigs/node-feature-discovery/pull/489